cluster/src/test/java/com/hartwig/pipeline/calling/sage/SageSomaticCallerTest.java

```diff
@@ -24,7 +24,7 @@ public void setUp() throws Exception {
 
     @Override
     protected Stage<SageOutput, SomaticRunMetadata> createVictim() {
-        return new SageSomaticCaller(TestInputs.defaultPair(), TestInputs.REF_GENOME_37_RESOURCE_FILES, new NoopPersistedDataset());
+        return new SageSomaticCaller(TestInputs.defaultPair(), TestInputs.REF_GENOME_37_RESOURCE_FILES, new NoopPersistedDataset(), false);
     }
 
```

Comment:
Can we add a test for this option? Something like PurpleTest.shallowModeUsesLowDepthSettings()